Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealityDeviceManager.cs

```diff
@@ -671,7 +717,14 @@ private BaseWindowsMixedRealitySource GetOrAddController(InteractionSource inter
             }
 
             string nameModifier = controllingHand == Handedness.None ? interactionSource.kind.ToString() : controllingHand.ToString();
+
+#if HP_CONTROLLER_ENABLED
+            bool isHPController = trackedMotionControllerStates.ContainsKey(controllerId) || !interactionSource.supportsTouchpad;
+            string inputSourceName = isHPController ? $"Mixed Reality Controller {nameModifier}" : $"HP Motion Controller {nameModifier}";
```

Comment:
This check looks backwards:

```suggestion
            string inputSourceName = isHPController ? $"HP Motion Controller {nameModifier}" : $"Mixed Reality Controller {nameModifier}";
```